js/mirador_strawberry.js

```diff
@@ -90,6 +90,14 @@
                                     }
                                 });
                             }
+                            let mirador_audios = document.querySelectorAll(".mirador-viewer audio source");
```

Comment:
Super cool! On a next iteration we can make a this condition

` if (mirador_videos.length ||  mirador_audios.length)`
and then a single `mutations.forEach(function (mutation)` (which is the most expensive for)
mutation.target.localName == "video" and mutation.target.localName == "audio" will work of course (or should?)

Just an idea for the future (or I will forget it)

Reply:
I was wondering if we could just expand the selector for both av types. Does this work (pardon my laughable jquery ignorance!!)?
```js
 let mirador_avs = document.querySelectorAll(".mirador-viewer video source, .mirador-viewer audio source");
```
then...
```js
if ($.inArray(mutation.target.localName, ["audio","video"]) && (mutation.addedNodes.length > 0) && (typeof(mutation.target.lastChild.src) != "undefined" )) {
```

Reply:
I guess so? What if I merge, and we try this first? =)